backpack/extensions/secondorder/diag_ggn/__init__.py

```diff
@@ -136,3 +136,83 @@ def __init__(self, mc_samples=1):
 
     def get_num_mc_samples(self):
         return self._mc_samples
+
+
+class BatchDiagGGN(BackpropExtension):
+    """
+    Individual Diagonal of the Generalized Gauss-Newton/Fisher.
+    Uses the exact Hessian of the loss w.r.t. the model output.
+
+    Stores the output in :code:`diag_ggn_exact_batch`,
+    has the same dimensions as the gradient.
```

Comment:
This statement is not correct. In fact, the output has the same dimension as the individual gradient. Can you look up the formulation of the output shape in the documentation of `BatchGrad` and insert it here?